dev-util/directx-headers/directx-headers-1.606.3.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN=DirectX-Headers
+inherit meson-multilib
+
+if [[ ${PV} == *9999* ]]; then
+	EGIT_REPO_URI="https://github.com/microsoft/${MY_PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/microsoft/${MY_PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
```

Comment:
You should only include keywords for architectures on which you have tested this package yourself (e.g. only `KEYWORDS="~amd64"`).